<!DOCTYPE html>
<html lang="en" xmlns="http://www.w3.org/1999/xhtml"
      xmlns:th="http://www.thymeleaf.org">
<head>
    <title th:text="${@siteSettings.title + ' - About'}">About</title>
    <object th:include="fragments/head :: head" th:remove="tag"/>
</head>
<body>
<div th:replace="fragments/navbar :: navbar(about)">...</div>
<div class="container-fluid p-4">

    <div th:replace="fragments/title :: title(title='About')" th:remove="tag">...</div>

    <div class="main-content p-4">
        <p> MaGS takes a number of protein features; including protein abundance, number of annotated interaction partners, number of annotated phosphorylation sites, and others; and combines them in a general linearized model to generate a score that ranks proteins for their propensity to be contained within biological condensates. Additionally, the server contains a new method, MaGSeq, which also is a general linearized model for protein condensate incorporation based only on protein sequence features without any other a priori knowledge of that protein’s biological context.  In MaGSeq, a number of other predictors, such as DISOPRED3 for protein intrinsic disorder and RPBBind for RNA-binding propensity, are used. To parameterize and test the model, proteins localizing to cellular condensates/granules were compiled from mass-spec and colocalization immunofluorescence experiments.  These ‘granulomes’ were then interrogated against cytoplasmic and proteomic controls for enriched protein features, which were then examined with PCA and QDA methods to find those features which best separated granulome proteins from the controls.  These features were then used in a quasibinomial GLM which underwent 10-fold cross validation during parameterization and was then tested against an independent validation set. </p>
	<p>Stress granule proteins used in model parameterization were taken from a series of publications detailed in the original MaGS manuscript (<a target="_blank" href="https://doi.org/10.1016/j.jmb.2020.02.020">doi: 10.1016/j.jmb.2020.02.020</a>). Briefly, these proteins (457 human and 321 yeast) were largely identified using mass spectrometry under heat or chemical stress and were curated based on the level of confidence from these publications.</p>
    </div>

    <div th:replace="fragments/title :: title(title='Citation')" th:remove="tag">...</div>
    <div class="main-content p-4">
        <p>Kuechler ER, Jacobson M, Mayor T, Gsponer J. GraPES: The granule protein enrichment server. in press.</p>
    </div>

    <div th:replace="fragments/title :: title(title='References')" th:remove="tag">...</div>

    <div class="main-content p-4">
        <p class="mb-0"><b>MaGS:</b></p>
        <p class="mb-2 ml-3">Kuechler ER, Budzyńska PM, Bernardini JP, Gsponer J, Mayor T. Distinct Features of Stress Granule Proteins Predict Localization in Membraneless Organelles. <em>J Mol Biol</em>. 2020 Mar 27;432(7):2349-2368. <a target="_blank" href="https://doi.org/10.1016/j.jmb.2020.02.020">doi: 10.1016/j.jmb.2020.02.020</a>. Epub 2020 Feb 24. PMID: 32105731.</p>
        <p class="mb-0"><b>DISOPRED3:</b></p>
        <p class="mb-2 ml-3">Jones DT, Cozzetto D. DISOPRED3: precise disordered region predictions with annotated protein-binding activity. <em>Bioinformatics</em>. 2015 Mar 15;31(6):857-63. <a target="_blank" href="https://doi.org/10.1093/bioinformatics/btu744">doi: 10.1093/bioinformatics/btu744</a>. Epub 2014 Nov 12. PMID: 25391399; PMCID: PMC4380029.</p>
        <p class="mb-0"><b>Camsol:</b></p>
        <p class="mb-2 ml-3">Sormanni P, Aprile FA, Vendruscolo M. The CamSol method of rational design of protein mutants with enhanced solubility. <em>J Mol Biol</em>. 2015 Jan 30;427(2):478-90. <a target="_blank" href="https://doi.org/10.1016/j.jmb.2014.09.026">doi: 10.1016/j.jmb.2014.09.026</a>. Epub 2014 Oct 14. PMID: 25451785.</p>
        <p class="mb-0"><b>Soluprot:</b></p>
        <p class="mb-2 ml-3">Musil, M., Konegger, H., Hon, J., Bednar, D., Damborsky, J. (2019). Computational Design of Stable and Soluble Biocatalysts. <em>ACS Catalysis</em> 9: 1033−1054.</p>
        <p class="mb-0"><b>Abundance:</b></p>
        <p class="mb-2 ml-3">Wang M, Herrmann CJ, Simonovic M, Szklarczyk D, von Mering C. Version 4.0 of PaxDb: Protein abundance data, integrated across model organisms, tissues, and cell-lines. <em>Proteomics</em>. 2015 Sep;15(18):3163-8. <a target="_blank" href="https://doi.org/10.1002/pmic.201400441">doi: 10.1002/pmic.201400441</a>. Epub 2015 Mar 12. PMID: 25656970; PMCID: PMC6680238.</p>
        <p class="mb-0"><b>PScore:</b></p>
        <p class="mb-2 ml-3">Vernon RM, Chong PA, Tsang B, Kim TH, Bah A, Farber P, Lin H, Forman-Kay JD. Pi-Pi contacts are an overlooked protein feature relevant to phase separation. <em>Elife</em>. 2018 Feb 9;7:e31486. <a target="_blank" href="https://doi.org/10.7554/eLife.31486">doi: 10.7554/eLife.31486</a>. PMID: 29424691; PMCID: PMC5847340.</p>
        <p class="mb-0"><b>RBPPred:</b></p>
        <p class="mb-2 ml-3">Zhang X, Liu S. RBPPred: predicting RNA-binding proteins from sequence using SVM. <em>Bioinformatics</em>. 2017 Mar 15;33(6):854-862. <a target="_blank" href="https://doi.org/10.1093/bioinformatics/btw730">doi: 10.1093/bioinformatics/btw730</a>. PMID: 27993780.</p>
        <p class="mb-0"><b>Tango:</b></p>
        <p class="mb-2 ml-3">Fernandez-Escamilla AM, Rousseau F, Schymkowitz J, Serrano L. Prediction of sequence-dependent and mutational effects on the aggregation of peptides and proteins. <em>Nat Biotechnol</em>. 2004 Oct;22(10):1302-6. <a target="_blank" href="https://doi.org/10.1038/nbt1012">doi: 10.1038/nbt1012</a>. Epub 2004 Sep 12. PMID: 15361882.</p>
        <p class="mb-0"><b>Phosphorylation Sites:</b></p>
        <p class="mb-2 ml-3">Huang KY, Lee TY, Kao HJ, Ma CT, Lee CC, Lin TH, Chang WC, Huang HD. dbPTM in 2019: exploring disease association and cross-talk of post-translational modifications. <em>Nucleic Acids Res</em>. 2019 Jan 8;47(D1):D298-D308. <a target="_blank" href="https://doi.org/10.1093/nar/gky1074">doi: 10.1093/nar/gky1074</a>. PMID: 30418626; PMCID: PMC6323979.</p>
	<p class="mb-0"><b>RNA-binding proteins:</b></p>
	<p class="mb-2 ml-3">Beckmann BM, Horos R, Fischer B, Castello A, Eichelbaum K, Alleaume AM, Schwarzl T, Curk T, Foehr S, Huber W, Krijgsveld J, Hentze MW. The RNA-binding proteomes from yeast to man harbour conserved enigmRBPs. <em>Nat Commun</em>. 2015 Dec 3;6:10127. <a target="_blank" href="https://doi.org/10.1038/ncomms10127">doi: 10.1038/ncomms10127</a>. PMID: 26632259; PMCID: PMC4686815.</p>
    </div>

    <div th:replace="fragments/title :: title(title='Terms & Conditions')" th:remove="tag">...</div>

    <div class="main-content p-4" >
        <p>This software is distributed on an as is basis, without any kind of warranties. This software is free for
            all. Appropriate citation when using <span th:text="${@siteSettings.title}" th:remove="tag">LIST-S2</span>
            in publication of scientific results. </p>
    </div>

    <div th:replace="fragments/title :: title(title='Browser Compatibility')" th:remove="tag">...</div>

    <div class="main-content p-4">
        <p>The latest versions of most browsers are supported.</p>
        <table class="table table-sm table-bordered table-striped">
            <thead>
                <tr>
                    <th>OS</th>
                    <th>Chrome</th>
                    <th>Firefox</th>
                    <th>Microsoft Edge</th>
                    <th>Safari</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <th scope="row">Windows</th>
                    <td class="text-success">Supported</td>
                    <td class="text-success">Supported</td>
                    <td class="text-success">Supported</td>
                    <td class="text-danger">Unsupported</td>
                </tr>
                <tr>
                    <th scope="row">Linux</th>
                    <td class="text-success">Supported</td>
                    <td class="text-success">Supported</td>
                    <td class="text-muted">N/A</td>
                    <td class="text-danger">Unsupported</td>
                </tr>
                <tr>
                    <th scope="row">Mac</th>
                    <td class="text-success">Supported</td>
                    <td class="text-warning">Untested</td>
                    <td class="text-muted">N/A</td>
                    <td class="text-success">Supported</td>
                </tr>
            </tbody>
        </table>
    </div>

</div>


    <div th:replace="fragments/footer" th:remove="tag">...</div>
</body>
</html>
